exchanges/ftx/ftx.go

```diff
@@ -1105,3 +1115,107 @@ func (f *FTX) GetOTCQuoteStatus(marketName, quoteID string) ([]QuoteStatusData,
 func (f *FTX) AcceptOTCQuote(quoteID string) error {
 	return f.SendAuthHTTPRequest(exchange.RestSpot, http.MethodPost, fmt.Sprintf(acceptOTCQuote, quoteID), nil, nil)
 }
+
+// GetSubaccounts returns the users subaccounts
+func (f *FTX) GetSubaccounts() ([]Subaccount, error) {
+	resp := struct {
+		Data []Subaccount `json:"result"`
+	}{}
+	return resp.Data, f.SendAuthHTTPRequest(exchange.RestSpot, http.MethodGet, subaccounts, nil, &resp)
+}
+
+// CreateSubaccount creates a new subaccount
+func (f *FTX) CreateSubaccount(name string) (*Subaccount, error) {
+	if name == "" {
+		return nil, errSubaccountNameMustBeSpecified
+	}
+	d := make(map[string]string)
+	d["nickname"] = name
+
+	resp := struct {
+		Data Subaccount `json:"result"`
+	}{}
+	if err := f.SendAuthHTTPRequest(exchange.RestSpot, http.MethodPost, subaccounts, d, &resp); err != nil {
+		return nil, err
+	}
+	return &resp.Data, nil
+}
+
+// UpdateSubaccountName updates an existing subaccount name
+func (f *FTX) UpdateSubaccountName(oldName, newName string) (*Subaccount, error) {
+	if oldName == "" || newName == "" || oldName == newName {
+		return nil, errSubaccountUpdateNameInvalid
+	}
+	d := make(map[string]string)
+	d["nickname"] = oldName
+	d["newNickname"] = newName
+
+	resp := struct {
+		Data Subaccount `json:"result"`
+	}{}
+	if err := f.SendAuthHTTPRequest(exchange.RestSpot, http.MethodPost, subaccountsUpdateName, d, &resp); err != nil {
+		return nil, err
+	}
+	return &resp.Data, nil
+}
+
+// DeleteSubaccountName deletes the specified subaccount name
+func (f *FTX) DeleteSubaccountName(name string) error {
```

Comment:
`DeleteSubAccountByName` or `DeleteSubAccount`? Since its actually deleting the account and not the name